prometheus_remediation/job_runner.py

```diff
@@ -137,12 +145,16 @@ def run_job_from_alert(event: PrometheusKubernetesAlert, params: JobParams):
     job.create()
 
     if params.notify:
-        event.add_enrichment([MarkdownBlock(f"Created job from alert: *{job_name}*.")])
+        event.add_enrichment([MarkdownBlock(f"Created Job from alert: *{job_name}*.")])
 
     if params.wait_for_completion:
-        wait_until_job_complete(job, params.completion_timeout)
-        job = hikaru.from_dict(job.to_dict(), cls=RobustaJob)  # temporary workaround for https://github.com/haxsaw/hikaru/issues/15
-        pod = job.get_single_pod()
-        event.add_enrichment([
-            FileBlock("job-runner-logs.txt", pod.get_logs())
-            ])
+        try:
+            wait_until_job_complete(job, params.completion_timeout)
+            job = hikaru.from_dict(job.to_dict(), cls=RobustaJob)  # temporary workaround for https://github.com/haxsaw/hikaru/issues/15
+            pod = job.get_single_pod()
+            event.add_enrichment([
+                FileBlock("job-runner-logs.txt", pod.get_logs())
+                ])
+        except Exception as e:
+            print(f"Timed out running {job_name} for alert {event.alert_name}")
```

Comment:
Use `logging.warning` not `print`.

Also, you're duplicating the string here and below. Make one copy, then log it and add it to the markdown block.

E.g:

```
err = f"...."
logging.warning(err)
event.add_enrichment([MarkdownBlock(err)])
```

As they say, DRY (don't repeat yourself)